src/main/java/org/variantsync/diffdetective/validation/ConstructionValidation.java

```diff
@@ -0,0 +1,450 @@
+package org.variantsync.diffdetective.validation;
```

Comment:
Very easy. Git actually handles this case because you renamed the whole directory.
Though, I created a new folder `experiments/thesis_bm` such that each paper/thesis has/becomes its own folder.